frontend/src/pages/Landing/index.js

```diff
@@ -191,17 +242,100 @@ function Landing() {
             <SimpleBar>
               <Container className="landing inline-size" padding={0}>
                 <Table className="usa-table usa-table--borderless usa-table--striped">
-                  <caption>Activity reports</caption>
+                  <caption>
+                    Activity reports
+                    <span className="smart-hub--table-nav">
+                      <span className="smart-hub--total-count">
+                        {renderTotal(offset, perPage, activePage, reportsCount)}
+                        <Pagination
```

Comment:
Putting this control in the caption makes it difficult-to-impossible to activate the control using VoiceOver.

Reply:
I tried putting it outside of the caption, but run into problems with stretching of the table. I can try again.

Reply:
Using Narrator on windows, I am able to hear and navigate to the "go to page x". Clicking on the link goes to the right page. We could put the navigation outside of the caption, but I found that (at least with Narrator) having the pagination inside of the caption element, works better. Wonder if the issue you are seeing is specific to the reader?

Reply:
It might be specific to the reader, but this isn't accessible by VoiceOver, so it needs to change. VoiceOver will read out the pagination details when it reads the table, but you cannot activate the links without some very creative back tabbing (which I could only figure out because I could see the table in the first place)

Reply:
Thanks! Yes, with a different screen reader (JAWS) I was able to replicate this behavior. The pagination control was moved out of the caption to reside before the table. I also added the aria-label.